MEDO v0.5.owl

```diff
@@ -804,6 +975,536 @@ LCME Glossary of Terms for LCME Accreditation Standards and Elements 2015-2016,
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/medo.owl#informal_summative_teacher_observation_and_feedback">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/MEDO_0000107"/>
+        <obo:IAO_0000115>A type of informal summative assessment where teacher or observer(s) observes, assesses, and provides written or oral subjective feedback of a learner&apos;s work or performance.</obo:IAO_0000115>
+        <obo:IAO_0000119>MedBiquitous - AM10 - Narrative Assessment: An instructor&apos;s or observer&apos;s written subjective assessment of a learner&apos;s work or performance (Mennin, McConnell, &amp; Anderson, 1997); May Include: Comments within larger assessment; Observation of learner or small group by instructor
+
+Does not include: Clinical Documentation Review; Clinical Performance Rating/Checklist; Peer Assessment; Self-Assessment</obo:IAO_0000119>
+        <rdfs:label>informal summative teacher observation and feedback</rdfs:label>
+        <skos1:related>MedBiquitous - AM10: Narrative Assessment</skos1:related>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/medo.owl#institutionally_developed_clinical_performance_exam -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/medo.owl#institutionally_developed_clinical_performance_exam">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/medo.owl#exam"/>
+        <obo:IAO_0000115>A type of formal summative practical performance-based examination developed internally for the measurement of problem-solving, clinical reasoning, decision making, and/or communication skills.</obo:IAO_0000115>
+        <obo:IAO_0000119>MedBiquitous - AM03: Practical performance-based examination developed internally to assess problem solving, clinical reasoning, decision making, and[/or] communication skills (Includes observation of learner or small group by instructor)
+
+Synonymous with/Includes: OSCE; Virtual Patient; Practical Exam; Internal (practical) Exam; Image Analysis; Script Concordance; Simulation Exam; MiniCEX; CEX; SCEE (Simulated Clinical Encounter Examination)</obo:IAO_0000119>
+        <rdfs:label>institutionally developed clinical performance exam</rdfs:label>
+        <skos1:related>MedBiquitous - AM03: Exam - Institutionally Developed, Clinical Performance</skos1:related>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/medo.owl#institutionally_developed_laboratory_practical_exam -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/medo.owl#institutionally_developed_laboratory_practical_exam">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/medo.owl#exam"/>
+        <obo:IAO_0000115>A type of formal summative examination that involves hands-on or simulated exercises in which students collect or use data to test and/or verify hypotheses or to address questions about principles and/or phenomena.</obo:IAO_0000115>
```

Comment:
Hate those and/or's: "A type of formal summative examination that involves hands-on or simulated exercises in which students collect or use data to test or verify hypotheses or to address questions about principles or phenomena."